src/ol/geom/linestring.js

```diff
@@ -203,6 +204,20 @@ ol.geom.LineString.prototype.intersectsExtent = function(extent) {
 
 
 /**
+ * @param {number} offset Offset.
+ * @param {number} count Count.
+ * @param {Array.<ol.Coordinate>=} opt_coordinates Coordinates.
+ * @api
+ */
+ol.geom.LineString.prototype.replaceCoordinates =
```

Comment:
This needs more documentation.  And I think the docs will probably need a couple examples - e.g. to remove coordinates, do ..., to add coordinates, do ..., etc.